scripts/ci/lib.sh

```diff
@@ -156,6 +156,12 @@ get_central_debug_dump() {
     ls -l "${output_dir}"
 }
 
+get_prometheus_metrics_parser() {
+    make prometheus-metric-parser -C "$ROOT"
+    export PATH="$ROOT/.gotools/bin":$PATH
```

Comment:
Can we prevent coupling here that hardcoding the path does?
Also we should probably quote $PATH.
How about something like:

```suggestion
    export PATH="$(dirname "$(make --quiet --whatever-flag-to-skip-messages-about-changing-dirs prometheus-metric-parser -C "$ROOT")"):$PATH"
```